src/main.c

```diff
@@ -172,18 +152,9 @@ int main(int argc, char **argv) {
 
     sqlite3 *db;
 
-    if(access(dbn, F_OK) != 0) {
-        if(init(dbn, &db) == false) {
-            fprintf(stderr, "Unable to initialize histx.db\n");
-        }
-    }
-    else {
-        int r = sqlite3_open(dbn, &db);
-        if (r) {
-            fprintf(stderr, "Unable to open %s [%s]\n", dbn, sqlite3_errmsg(db));
-            sqlite3_close(db);
-            return -1;
-        }
+    if(init(dbn, &db) == false) {
+        fprintf(stderr, "Unable to initialize histx.db\n");
+        return -1;
```

Comment:
The old code ran the "initialize db" code if no DB was found - otherwise it would simply open the DB.

Now, it runs the "initialize" code each time, and all `create` commands have become `create if not exist`